Source/WebInspectorUI/UserInterface/Views/DOMTreeElement.js

```diff
@@ -2214,6 +2217,14 @@ WI.DOMTreeElement = class DOMTreeElement extends WI.TreeElement
     {
         this._createBadges();
     }
+
+    // Popover delegate
```

Comment:
Style: delegate methods should be put after `// Public` and before `// Protected`

Reply:
Agreed. This file doesn't have a clear distinction between Public / Protected / Private members. I won't re-arrange those in this patch to keep a sane diff.